src/providers/google/chatComplete.ts

```diff
@@ -226,6 +226,28 @@ interface GoogleGenerateContentResponse {
   };
 }
 
+// this converts the google stop_reason to an openai finish_reason
+// https://ai.google.dev/api/python/google/ai/generativelanguage/Candidate/FinishReason
+const getFinishReason = (stopReason?: string): string | null => {
```

Comment:
Same comments as in the getFinishReason transformer function above.
1. Use switch/case
2. Define types for GoogleStopReason